src/features/people/PeopleDetails/index.js

```diff
@@ -18,20 +25,94 @@ const PersonDetails = () => {
 
     const details = useSelector(selectDetails);
     const status = useSelector(selectStatus);
+    const actorCast = useSelector(selectPersonCast);
+    const actorCrew = useSelector(selectPersonCrew);
+    const [genres, setGenres] = useState([]);
+
+    useEffect(() => {
+        const fetchGenres = async () => {
+            const genresData = await getGenreList();
+            setGenres(genresData.genres);
+        };
+        fetchGenres();
+    }, []);
+
+    const nameGenres = (genreIds) => {
+        if (!genres || genres.length === 0) {
```

Comment:
I like that you show that you handle the case when the genre array is empty.👏🏻 
It seems to me that the map method used on an empty array will return an empty array, so I wonder if this is necessary? 🤔

Reply:
Thanks! 😊 I'll simplify the code by omitting the check in my next PR. Appreciate your feedback and attention to detail! 🚀👍🏻